src/modules/pdt/doc/pdt_admin.xml

```diff
@@ -501,11 +501,8 @@ pd_translate("*", "$var(a)");
 		<para>
 		RPC Command Example:
 		</para>
-		<programlisting  format="linespecific">
-...
-# &kamcmd; _sdomain_ _prefix_ _domain_
-&kamcmd; kamailio.org 123
-...
```

Comment:
Thanks!
Makes sense, formatting restored in commit 06a0a1f6e91e4d1c0f87afb33519f1de4584a918